samples/nrf9160/azure_fota/Kconfig

```
@@ -14,4 +14,7 @@ module = AZURE_FOTA_SAMPLE
 module-str = Azure FOTA sample
 source "${ZEPHYR_BASE}/subsys/logging/Kconfig.template.log_config"
 
+config DEVICE_ID_USE_UUID
+	bool "Use the devices's modem UUID as device ID when connecting to Azure IOT hub"
```

I think this can be reduced to `Use the device's modem UUID as device ID` and than the full text can be copied over to the help text like this:
```suggestion
	bool "Use modem UUID as device ID"
	  help "Use the device's modem UUID as device ID when connecting to Azure IoT Hub"
```